src/components/Navbar.jsx

```diff
@@ -1,27 +1,54 @@
 // LIBRARY IMPORTS
-import React from 'react';
+import React, { useState } from 'react';
 import { NavLink } from 'react-router-dom';
 
 // LOCAL IMPORTS
 
 export default function NavBar() {
+  const clicked = {clickedVoiceIcon:false, clickedAboutIcon:false}
```

Comment:
great idea storing these states separately in an object to help with color change! common JS naming convention for booleans usually uses verb 'is' (e.g. `isClicked`)